build/DependencyVersions.props

```diff
@@ -3,15 +3,15 @@
   <PropertyGroup>
     <MicrosoftAspNetCoreAllPackageVersion>2.1.0-rtm-30752</MicrosoftAspNetCoreAllPackageVersion>
     <MicrosoftAspNetCoreAppPackageVersion>$(MicrosoftAspNetCoreAllPackageVersion)</MicrosoftAspNetCoreAppPackageVersion>
-    <MicrosoftNETCoreAppPackageVersion>2.1.0-rtm-26508-02</MicrosoftNETCoreAppPackageVersion>
+    <MicrosoftNETCoreAppPackageVersion>2.2.0-preview1-26515-04</MicrosoftNETCoreAppPackageVersion>
```

Comment:
I don't think you want this, right?  My understanding is the `2.1.4xxx` version of the SDK will depend on/include the `2.1.x` version of the runtime.  You don't want `2.2.0`.